src/main/java/rx/observers/SafeCompletableSubscriber.java

```diff
@@ -52,8 +52,8 @@ public void onCompleted() {
 
     @Override
     public void onError(Throwable e) {
+        RxJavaPluginUtils.handleException(e);
```

Comment:
Do you want me to make a unit test for the case that `onError` gets called twice in a `SafeSubcriber` and `SafeCompletableSubscriber` to see if the error handler gets called twice? I don't know if it's worth it. What it happens is that it just doesn't look good to me when behavior is different in `Completable` from `Single` and `Observable`.